iOSClient/Trash/NCTrash+CollectionView.swift

```diff
@@ -84,7 +84,7 @@ extension NCTrash: UICollectionViewDataSource {
             cell = listCell
         } else {
             // GRID
-            guard let gridCell = collectionView.dequeueReusableCell(withReuseIdentifier: "gridCell", for: indexPath) as? NCGridCell else { return UICollectionViewCell() }
+            guard let gridCell = collectionView.dequeueReusableCell(withReuseIdentifier: "gridCell", for: indexPath) as? NCTrashGridCell else { return UICollectionViewCell() }
```

Comment:
Suggestion: Wherever existing code is customized. Put a comment above mentioning "MagentaCloud Customization" to identify our customization during code conflict. It will also help new developers.